lib/cli/ui/progress.rb

```diff
@@ -19,14 +19,14 @@ class Progress
       #     bar.tick(set_percent: percent)
       #   end
       #
-      # Increase the percent by 1 percent
+      # Increase the percent by 0.01 percent
```

Comment:
This does actually increase by 1 percent. Can you drop this change?

Reply:
Sure 👍